vuepress/docs/next/docs/reference/deployment-structure.md

```diff
@@ -7,15 +7,15 @@ sidebarDepth: 2
 This page provides an overview of the key Entando GitHub repositories with brief descriptions 
 of how these repositories are realized in a running Entando deployment. It also explores the architecture behind the Entando Platform and how its functions are structured.
 
-## entando-operator
-The Entando Operator coordinates the installation and configuration of all the components in an Entando 
-cluster. The operator can be installed once per cluster and is used to coordinate the plugin lifecycle for multiple Entando Applications across many namespaces.
+## Entando Operator
+The **Entando Operator** coordinates the installation and configuration of all the components in an Entando 
+cluster. The operator can be installed once per cluster, and is used to coordinate the plugin lifecycle for multiple Entando Applications across many namespaces.
 
 * GitHub: <https://github.com/entando-k8s/entando-k8s-controller-coordinator/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-k8s-controller-coordinator>
 
 #### Customization
-Use existing [Entando Custom Resources](../consume/custom-resources.md) to extend the Entando Platform.
+Use the [Entando Custom Resources](../consume/custom-resources.md) to extend the Platform.
```

Comment:
custom resources should not be capitalized outside of a title; although you're linking to a title the capitalization here implies the term is always capitalized

Reply:
platform should not be capitalized unless it directly follows "Entando" (per our glossary)

use the Entando Custom Resources -> use Entando Custom Resources -- having "the" there makes it seem like you need to use all of them and can't pick and choose